Sanity/pkcs11/basic/Makefile

```diff
@@ -0,0 +1,64 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Makefile of /CoreOS/clevis/Sanity/pkcs11/basic
+#   Description: tests the basic pkcs#11 functionality of clevis
+#   Author: Martin Litwora <mlitwora@redhat.com>
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Copyright (c) 2017 Red Hat, Inc.
```

Comment:
Wrong year (2024)

Reply:
Great catches, thanks!

Reply:
fixed in the https://github.com/RedHat-SP-Security/clevis-tests/pull/8/commits/0711f411e56b6eca521842ef1fb393ceefd08eea